base_layer/wallet/migrations/2022-08-08-134037_initial/up.sql

```diff
@@ -39,75 +42,88 @@ CREATE TABLE inbound_transactions (
+    commitment                              BLOB                NULL,
+    spending_key                            BLOB                NOT NULL,
+    value                                   BIGINT              NOT NULL,
+    output_type                             INTEGER             NOT NULL,
+    maturity                                BIGINT              NOT NULL,
+    status                                  INTEGER             NOT NULL,
+    hash                                    BLOB                NULL,
+    script                                  BLOB                NOT NULL,
+    input_data                              BLOB                NOT NULL,
+    script_private_key                      BLOB                NOT NULL,
+    script_lock_height                      UNSIGNED BIGINT     NOT NULL DEFAULT 0,
+    sender_offset_public_key                BLOB                NOT NULL,
+    metadata_signature_ephemeral_commitment BLOB                NOT NULL,
+    metadata_signature_ephemeral_pubkey     BLOB                NOT NULL,
+    metadata_signature_u_a                  BLOB                NOT NULL,
+    metadata_signature_u_x                  BLOB                NOT NULL,
+    metadata_signature_u_y                  BLOB                NOT NULL,
+    mined_height                            UNSIGNED BIGINT     NULL,
+    mined_in_block                          BLOB                NULL,
+    mined_mmr_position                      BIGINT              NULL,
+    marked_deleted_at_height                BIGINT              NULL,
+    marked_deleted_in_block                 BLOB                NULL,
+    received_in_tx_id                       BIGINT              NULL,
+    spent_in_tx_id                          BIGINT              NULL,
+    coinbase_block_height                   UNSIGNED BIGINT     NULL,
+    metadata                                BLOB                NULL,
+    features_json                           TEXT                NOT NULL DEFAULT '{}',
+    spending_priority                       UNSIGNED INTEGER    NOT NULL DEFAULT 500,
+    covenant                                BLOB                NOT NULL,
+    mined_timestamp                         DATETIME            NULL,
+    encrypted_value                         BLOB                NOT NULL,
+    minimum_value_promise                   BIGINT              NOT NULL,
+    source                                  INTEGER             NOT NULL DEFAULT 0,
     CONSTRAINT unique_commitment UNIQUE (commitment)
 );
 
-CREATE TABLE scanned_blocks (
+CREATE TABLE scanned_blocks
+(
     header_hash BLOB PRIMARY KEY NOT NULL,
     height      BIGINT           NOT NULL,
     num_outputs BIGINT           NULL,
     amount      BIGINT           NULL,
     timestamp   DATETIME         NOT NULL
 );
 
-CREATE TABLE wallet_settings (
+CREATE TABLE wallet_settings
+(
     key   TEXT PRIMARY KEY NOT NULL,
     value TEXT             NOT NULL
 );
+
+CREATE TABLE burnt_proofs
```

Comment:
This will break all existing wallets, and create a proper migration here

Leave this file as is

Reply:
you're right, I'll create a new migration for that